app/src/main/java/backgammon/Move.java

```diff
@@ -0,0 +1,63 @@
+package backgammon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Move {
+
+  private final int from;
+  private final int to;
+  private final List<Movelet> movelets = new ArrayList<>();
+
+
+  public Move(int from, int to, List<Integer> dice, boolean isDoubles) {
+    this.from = from;
+    this.to = to;
+
+    int die = dice.get(0);
+
+    for (int i = 0; i < dice.size() && isDoubles; i++) {
+      movelets.add(new Movelet(from - die * i, from - die * (i + 1), die ));
+    }
+
+    if (!isDoubles) {
+      movelets.add(new Movelet(from, from - die, die ));
+      if (dice.size() == 2)
+        movelets.add(new Movelet(from - die, from - die - dice.get(1), dice.get(1)));
+    }
+  }
+
+
+  //Will need this to remove the dice from the list of dice in the game loop
+  public List<Integer> getDice(List<Integer> dice) {
+    for (Movelet movelet : movelets)
+      dice.add(movelet.getDie());
+    return dice;
+  }
```

Comment:
I'm still confused about why you're needing to pass in dice to getDice here.
Also you should not be mutating state in a getter function.

Reply:
The "dice" list is just an empty list that the used dice would be added to. We'll have to remove the dice that have been used from wherever they're contained, so we need a way to find out which ones were used. I should probably rename it to usedDice and move it into the method. I just figured if I put it there it'd save 1 line.